sites/vital-demo/.storybook/main.js

```diff
@@ -24,5 +24,6 @@ module.exports = {
   ],
   "core": {
     "builder": "webpack5"
-  }
+  },
+  "staticDirs": [{ from: '../node_modules/', to: './src/css' }],
```

Comment:
From sites/vital-demo
—> nom run storybook
info => Loading presets
info => Serving static files from ././node_modules/ at /./src/css
…
so in theory its mapping 'sites/vital-demo/node_modules' to 'sites/src/css' ????? 

ERROR from storybook: relative path: ./src/css/@bodiless/vital-link/assets/font/linkicons.woff2 that storybook expects….

PATH exists from root: 
sites/vital-demo/node_modules/@bodiless/vital-link/assets/font/linkicons.woff2